src/components/data_search/DatasetFilterList.tsx

```diff
@@ -20,7 +24,16 @@ export const FilterItemHeader = (props) => {
   );
 };
 
-export const FilterItemList = (props) => {
+interface FilterItemListProps {
+  category: string;
+  datasets: DatasetTerm[]
```

Comment:
This declares `datasets` but this prop is not extracted below. Is the idea that it will be needed in the future?